### ISSUE TITLE
Implementing STM cp times to race directly against

### ISSUE THREAD

**Issue by enderxdd** [jamesy012/OpenPlanet-BestCheckpoints#19]
I had the idea to implement a functionnality that shows the cps from the STMs, and your delta to those cps, that way you can indirectly "race" the STM ghosts and see how far ahead/behind you are during your run
I'd be able to provide the STM cp times (extracted with bigbang's tool) for them to be then implemented via the plugin, available for contact on discord (ender#8128)

**Comment by jamesy012** [comment]
yeah ive been thinking of adding something to compare to other people
mainly against the ghosts so far, but they seem tricky to deal with and the online players dont give me the data I want

I'll give you a message later on
